docs/sidebar.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * this script sets the slack github action docs sidebar 
+ * 
+ * sidebar values take the form of "slack-github-action/path-within-docs/"
+ * in other words - full path but remove "docs"
+ *
+ * for info on syntax see https://docusaurus.io/docs/sidebar
+ *
+ * this file is copied into slackapi.gibhub.io/slack-github-action/sidebar.js 
+ * it is then called in slackapi.github.io/sidebars.js
+ */
+
+export default [
+  {
+    type: 'doc',
+    id: 'slack-github-action/slack-github-action',
+    label: 'Slack GitHub Action',
+    className: 'sidebar-title',
+  },
+  { type: 'html', value: '<hr>' },
+  'slack-github-action/sending-variables',
+  {
+    type: 'category',
+    label: 'Sending techniques',
+    collapsed: false,
+    link: {
+      type: 'doc',
+      id: 'slack-github-action/sending-techniques/sending-techniques',
+    },
```

Comment:
🔗 Ooh this is fascinating to me:

https://github.com/slackapi/slack-github-action/blob/c69deab25713549329730019e9c20a81d09bb4cd/docs/sending-techniques/sending-techniques.md?plain=1#L1-L5